src/modules/utils/motordrivercontrol/MotorDriverControl.cpp

```diff
@@ -153,32 +157,52 @@ bool MotorDriverControl::config_module(uint16_t cs)
     initialize_chip(cs);
 
     // if raw registers are defined set them 1,2,3 etc in hex
-    str= THEKERNEL->config->value( motor_driver_control_checksum, cs, raw_register_checksum)->by_default("")->as_string();
-    if(!str.empty()) {
-        rawreg= true;
-        std::vector<uint32_t> regs= parse_number_list(str.c_str(), 16);
-        if(!regs.empty()) {
-            uint32_t reg= 0;
-            for(auto i : regs) {
-                // this just sets the local storage, it does not write to the chip
-                switch(chip) {
-                    case DRV8711: drv8711->set_raw_register(&StreamOutput::NullStream, ++reg, i); break;
-                    case TMC2660: tmc26x->setRawRegister(&StreamOutput::NullStream, ++reg, i); break;
-                }
-            }
-
-            // write the stored registers
-            switch(chip) {
-                case DRV8711: drv8711->set_raw_register(&StreamOutput::NullStream, 255, 0); break;
-                case TMC2660: tmc26x->setRawRegister(&StreamOutput::NullStream, 255, 0); break;
-            }
-        }
-
-    }else{
-        rawreg= false;
-    }
+	rawreg=false;
```

Comment:
Please avoid whitespace changes, it is annoying and makes it hard to review.

Reply:
This was not a whitespace change.

I moved `rawreg=false;` up to the top of the "rawreg" area in the config module, since I added further "rawreg" code right after it.  It made sense, at the time, moving it up there as I was adding my additional rawreg config code, in which I also set `rawreg=true;`.  Looking at how the code turned out, it would have worked the same where it was, but all things considered, it is better where it is now, yes?  (Not even sure what purpose rawreg flag serves, but included it in my code for consistency since it is also writing raw register data to the drivers.)